flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -455,6 +457,94 @@ public void createTable(CatalogBaseTable table, ObjectIdentifier objectIdentifie
 			"CreateTable");
 	}
 
+	/**
+	 * Creates a database in a given fully qualified path.
```

Comment:
Nit: an empty line below this to be consistent with existing comment style. Same for other newly added methods.

Also, we should give description to the parameters.